tests/ImageSharp.Drawing.Tests/Drawing/Text/DrawTextOnImageTests.cs

```diff
@@ -201,6 +201,51 @@ public void OpenSansJWithNoneZeroShouldntExtendPastGlyphe<TPixel>(TestImageProvi
                 comparer,
                 img => img.Mutate(c => c.DrawText(textOptions, sb.ToString(), font, color, new PointF(10, 1))),
                 false,
+                true);
+        }
+
+        [Theory]
+        [WithSolidFilledImages(400, 600, "White", PixelTypes.Rgba32, 1, 5)]
+        [WithSolidFilledImages(400, 600, "White", PixelTypes.Rgba32, 1.5, 3)]
+        [WithSolidFilledImages(400, 600, "White", PixelTypes.Rgba32, 2, 2)]
+        public void FontShapesAreRenderedCorrectly_WithLineSpacing<TPixel>(
+            TestImageProvider<TPixel> provider,
+            float lineSpacing,
+            int lineCount)
+            where TPixel : unmanaged, IPixel<TPixel>
+        {
+            Font font = CreateFont("OpenSans-Regular.ttf", 16);
+
+            var sb = new StringBuilder();
+            string str = "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa. Fusce posuere, magna sed pulvinar ultricies, purus lectus malesuada libero, sit amet commodo magna eros quis urna.";
```

Comment:
```suggestion
            string str = "We want ... a SHRUBBERY!";
```

By `lineSpacing` I actually meant the total number of lines (text rows), to stress the corner cases of 1 and 2 lines.

Reply:
Hold on, I don't follow here... I thought we were testing to ensure that multiple individual text blocks wrap appropriately and have the correct leading space?

`lineSpacing` repesents the `LineSpacing` line height multiplier. `lineCount` represents the number of lines to render separated by a new line character.

Reply:
Sorry mistyped:

> By ~lineSpacing~ `lineCount` I actually meant the total number of lines (text rows), to stress the corner cases of 1 and 2 lines.~

Reply:
Ah... So you want to see a test without wrapping text? I'm not sure we'll gain much insight there. 

I'd have to measure the text against the bounds of the output in order to gauge wrapping correctly. That's more complicated.

Reply:
It's not about the wrapping. I thought rendering 1, 2 and *a few* lines is enough to test the feature. It seemed irrelevant whether it's manual break or automatic wrapping.

Checking `TextLayout.cs` again, looks like automatic wrapping is relevant here, and we need to test that as well.

I leave this to your judgment.

Reply:
Yeah a manual break resets the location edge of the line and affects glyph location properties. That's why I'm explicitly setting a wrapping width in the tests since it will simply be cutoff otherwise.

https://github.com/SixLabors/Fonts/blob/2974546fb30dfdb4e18b8443c8af10d1353c246d/src/SixLabors.Fonts/TextLayout.cs#L225-L247

The leading line height of the first line is also reset to undo the scaling. 

https://github.com/SixLabors/Fonts/blob/2974546fb30dfdb4e18b8443c8af10d1353c246d/src/SixLabors.Fonts/TextLayout.cs#L123-L136

I'm inclined to leave the test for now. I think text layout could do with some optimization love so we can revisit the tests then.